dlp/api/Snippets/InspectStorageWithSCCIntegration.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_inspect_gcs_send_to_scc]
+
+using System.Collections.Generic;
+using System.Linq;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using static Google.Cloud.Dlp.V2.InspectConfig.Types;
+
+public class InspectStorageWithSCCIntegration
+{
+    public static DlpJob SendGcsData(
+        string projectId,
+        string gcsPath,
+        Likelihood minLikelihood = Likelihood.Unlikely,
+        IEnumerable<InfoType> infoTypes = null)
+    {
+        // TODO
```

Comment:
Clean up required?

Reply:
Sorry my bad. Forgot to remove comment. Thanks.